lib/graph_ql/fields/abstract_field.rb

```diff
@@ -7,20 +7,19 @@
 #   - #deprecation_reason
 class GraphQL::AbstractField
   def name
-    raise NotImplementedError, "#{self.class.name}#name should return the name for accessing this field"
+    fail NotImplementedError, "#{self.class.name}#name should return the name for accessing this field"
   end
 
   def type
-    raise NotImplementedError, "#{self.class.name}#type should return the type class which this field returns"
+    fail NotImplementedError, "#{self.class.name}#type should return the type class which this field returns"
   end
 
   def description
-    raise NotImplementedError, "#{self.class.name}#description should return this field's description"
+    fail NotImplementedError, "#{self.class.name}#description should return this field's description"
```

Comment:
meh, this class was a terrible idea anyways, should have saved your time. i'll yank it out of here soon